ui_framework/src/components/popover/_popover.scss

```diff
@@ -1,7 +1,8 @@
 // Pop menu is an animated popover relatively positioned to a button / action.
 // By default it positions in the middle, but can be anchored left and right.
 
-@include component('kuiPopMenu') {
+@include component('kuiPopover') {
```

Comment:
Discussed on slack. Modifier for less common case (the header) to make it flush.